src/authenticator/basicUrlAuthenticator.js

```diff
@@ -15,6 +16,10 @@ function BasicUrlAuthenticator(config,instanceConfig,logger){
 
   this.user = instanceConfig.user;
   this.pass = instanceConfig.pass;
+  
+  if (config.authOnce) {
```

Comment:
why do you need to implement it in every authenticator, why not only centrally?

Reply:
Can you please share UIveri5 developer document venkatkccs53@gmail.com